stylesheets/_hacks.scss

```diff
@@ -106,3 +106,8 @@ blockquote p {
 .plain {
   text-decoration: none;
 }
+
+
+.floating:before {
+  margin-right: 0!important; // XXX JAMES SAID THIS IS OKAY
```

Comment:
Is this okay to add to all floating elements? I'm sure we are using that that class in more places than the nav. Maybe we should apply this only to `.floating` inside the nav?